### ISSUE TITLE
A11y: display close button in the menu panel

### ISSUE THREAD

**Issue by audrasjb** [WordPress/twentytwenty#27]
I think there should be a close button in the menu panel, it would be useful for users that use their keyboard to navigate through the content of the website.

I know the menu button toggles to a close button when it's clicked, but the toggled close button should probably be displayed on the menu panel so the user could see the focus is on a close button :-)

**Comment by andersnoren** [comment]
That will definitely be included. The current mockup for the menu on desktop looks like this:

<img width="1440" alt="Menu → Desktop" src="https://user-images.githubusercontent.com/8181091/64465408-cff0fa80-d10c-11e9-9255-f1e147dd0ea6.png">

**Comment by audrasjb** [comment]
Yeah, great news!
Thanks for the feedback 👍
